lib/chargify_api_ares/resources/component_price_point.rb

```diff
@@ -0,0 +1,37 @@
+module Chargify
+  class ComponentPricePoint < Base
+    self.collection_parser = ::ComponentPricePointCollection
+
+    self.prefix = '/components/:component_id/'
+    self.collection_name = 'price_points'
+
+    def self.all(component_id)
+      find(:all, { params: { component_id:, 'filter[type]' => 'catalog,default,custom' } })
+    end
+
+    def self.find_subscription_price_point(component_id, subscription_id)
+      price_points = all(component_id)
+      subscription_price_point = find_by_attribute(price_points, :subscription_id, subscription_id)
+
+      return subscription_price_point if subscription_price_point
```

Comment:
what if there are more than one?

Reply:
I need to change it to return the last one instead doing `find`

Reply:
Updated!